backend/src/schema/resolvers/statistics.js

```diff
@@ -33,10 +33,10 @@ export default {
          * Note: invites count is calculated this way because invitation codes are not in use yet
          */
         response.countInvites = response.countEmails - response.countUsers
+        return response
       } finally {
         session.close()
       }
-      return response
     },
   },
 }
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T13:12:42Z" title="Tuesday, December 3rd 2019, 2:12:42 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -33,10 +33,10 @@ export default {
          * Note: invites count is calculated this way because invitation codes are not in use yet
          */
         response.countInvites = response.countEmails - response.countUsers
+        return response
       } finally {
         session.close()
       }
-      return response
     },
   },
 }
```


same as :point_up_2: 
would give greater confidence if we had a test suite for this resolver